ProfileLikelihoodRatio.py

```diff
@@ -80,7 +80,7 @@ def freeze_target_parameter(frozen: bool) -> None:
             Args:
                 frozen: If true, the target parameter is frozen in the fit.
             """
-            for model in dataset.models:
+            for model in datasets.models:
                 if model.name != self.limit_target.model.name:
```

Comment:
could be simplified (here in other places)
`model = datasets.models[self.limit_target.model.name]`

Reply:
same for parameters few line below
`parameter = model.parameters["self.limit_target.parameter_name"]`